### PR TITLE
python for example 2

### PR DESCRIPTION
**Pull request by atrisovic**
some of the things I didn't know how to translate :-s

### CODE REVIEW

File: tutorial-content/content/example-step2.md [hunk 1]
```diff
@@ -55,6 +56,29 @@ writeRaster(rr3, "../data/pcount/usap90ag.nc4",
...
 ```
+````
+ 
+````{tabbed} Python
+```{code-block} python
+#!pip install rasterio # will work in Jupyter Notebook
+import xarray as xr
+rr = xr.open_rasterio("../data/pcount/usap90ag.bil")
+
+#TODO rr2 <- aggregate(rr, fact=24, fun=sum)
```
Comment (jrising) [inline tutorial-content/content/example-step2.md:67]: One solution here is to generate an appropriately sized numpy zeros matrix and then iterate through np.arange(ii, N, 24) in x and y, summing each into the matrix.

**Review by jrising — COMMENTED** [review]
Do you want me to write in the brute-force method of doing aggregation?

**Comment by atrisovic** [comment]
That would be much appreciated! I didn't know how to do it. I'm also not sure if I translated all the instructions from the second code block. 
I left the markdown tables to make sure it looks as it should, but we could also move them after the code block box for people to check (this one was helpful for making markdown tables in python: `!pip install tabulate` and `print(df.to_markdown()`)). 
I will now do a read through, and maybe we can plan to ping Julia in a few days or a week if that makes sense.

**Comment by jrising** [comment]
Okay, I've gone through the code and made sure that it produces results that match the R code. It turns out the `xarray` has a function that's equivalent to R's `aggregate` function: `coarsen`.
https://github.com/atrisovic/weather-panel.github.io/commit/d503c05a6f62f1a0e8d66af2159dd18c4395288e

Feel free to merge!

**Comment by atrisovic** [comment]
That's great! Thank you!

**Pull request merged**